client/my-sites/site-settings/media-settings/index.jsx

```diff
@@ -0,0 +1,114 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormSelect from 'components/forms/form-select';
+import FormLabel from 'components/forms/form-label';
+import FormCheckbox from 'components/forms/form-checkbox';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import { protectForm } from 'lib/protect-form';
+import { isJetpackModuleActive } from 'state/selectors';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { updateSettings } from 'state/jetpack/settings/actions';
+
+class MediaSettings extends Component {
+
+	onInputChange = ( updated ) => {
+		this.setState( updated );
+		this.props.markChanged();
+	}
+
+	render() {
+		const props = this.props;
+		return (
+			<Card className="media-settings site-settings">
+				<FormFieldset>
+					<div className="media-settings__info-link-container">
+						<InfoPopover position={ 'left' }>
+							<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/support/photon' } >
+								{ props.translate( 'Learn more about Photon' ) }
+							</ExternalLink>
+						</InfoPopover>
+					</div>
+					<JetpackModuleToggle
+						siteId={ props.siteId }
+						moduleSlug="photon"
+						label={ props.translate( 'Speed up your images and photos with Photon.' ) }
+						description="Enabling Photon is required to use Tiled Galleries."
+						disabled={ props.isRequestingSettings || props.isSavingSettings }
+						/>
+				</FormFieldset>
+				<FormFieldset className="media-settings__formfieldset has-divider is-top-only">
+					<div className="media-settings__info-link-container">
+						<InfoPopover position={ 'left' }>
+							<ExternalLink target="_blank" icon={ true } href={ 'https://jetpack.com/support/carousel' } >
+								{ props.translate( 'Learn more about Carousel' ) }
+							</ExternalLink>
+						</InfoPopover>
+					</div>
+					<JetpackModuleToggle
+						siteId={ props.siteId }
+						moduleSlug="carousel"
+						label={ props.translate( 'Transform image galleries into full screen slideshows.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **6** times:
`translate( 'Transform standard image galleries into full-screen slideshows.' )` *ES Score: 17.67*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).